packages/react-openapi/src/ScalarApiButton.tsx

```diff
@@ -98,7 +98,10 @@ export function ScalarApiClient(props: { children: React.ReactNode }) {
             {
                 url: operationData.path,
             },
-            getRequestFromOperation(operation, { requiredOnly: false }),
+            getRequestFromOperation(
+                { ...operation, information: operationData.operation },
```

Comment:
@marclave There might be some incompleteness on typing here: 

<img width="1426" alt="Screenshot 2024-05-01 at 15 25 54" src="https://github.com/GitbookIO/gitbook/assets/701222/1703c9b7-b121-4f06-b2d4-ff37639568cb">

Reply:
ah! let me fix this : )

Reply:
https://github.com/GitbookIO/gitbook/pull/2310/files

fixed :)